backend/src/main/java/pt/ua/deti/ies/SmartHomes/backend/WebSocket/WebSocketConfig.java

```diff
@@ -0,0 +1,24 @@
+package pt.ua.deti.ies.SmartHomes.backend.WebSocket;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.messaging.simp.config.MessageBrokerRegistry;
+import org.springframework.web.socket.config.annotation.EnableWebSocketMessageBroker;
+import org.springframework.web.socket.config.annotation.StompEndpointRegistry;
+import org.springframework.web.socket.config.annotation.WebSocketMessageBrokerConfigurer;
+
+@Configuration
+@EnableWebSocketMessageBroker
+public class WebSocketConfig implements WebSocketMessageBrokerConfigurer {
+
+  @Override
+  public void configureMessageBroker(MessageBrokerRegistry config) {
+    config.enableSimpleBroker("/topic");
```

Comment:
Should it be `"/electricity"`? Or as I suggested in another comment, `"/0/electricity"`?